DEVELOPMENT.md

```diff
@@ -1,10 +1,10 @@
 # How to Develop TensorBoard
 
-TensorBoard at HEAD relies on the nightly installation of TensorFlow, so please install TensorFlow nightly for development. To install TensorFlow nightly, `pip install` the link to the [appropriate whl file listed at the TensorFlow repository](https://github.com/tensorflow/tensorflow).
+TensorBoard at HEAD relies on the nightly installation of TensorFlow: this allows plugin authors to use the latest features of TensorFlow, but it means release versions of TensorFlow may not suffice for development. We recommend installing TensorFlow nightly in a [Python virtualenv](https://virtualenv.pypa.io), and then running your modified development copy of TensorBoard within that virtualenv. To install TensorFlow nightly within the virtualenv, you can simply run `pip install tf-nightly`.
```

Comment:
How about: "To install TensorFlow nightly within a virtualenv, simply run:

```sh
virtualenv foo
source foo/bin/activate
pip install --upgrade pip
pip install tf-nightly
```

Reply:
Done.  Also added prompt indicators for which commands should be in the virtualenv.